modules/images-other-jenkins-memory.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins.adoc
+
+[id="images-other-jenkins-memory_{context}"]
+= Jenkins memory requirements
+
+When deployed by the provided Jenkins Ephemeral or Jenkins Persistent
+templates, the default memory limit is 512MiB.
+
+For memory efficiency, by default the Jenkins image dynamically uses a 32-bit
```

Comment:
Should remove this section too.

Reply:
Commented out for now so it won't build. Should the whole module be removed or just the JVM content?

Reply:
IMO, only JVM content